pipeline/cron-pipeline-test-executor.groovy

```diff
@@ -0,0 +1,211 @@
+/*
+    Pipeline script for executing long running cephci test suites.
+*/
+
+def nodeName = "centos-7"
+def testStages = [:]
+def testResults = [:]
+def rhcephVersion
+def buildArtifacts
+def buildType
+def buildPhase
+def sharedLib
+
+
+node(nodeName) {
+    try {
+        timeout(unit: "MINUTES", time: 30) {
+            stage('Install prereq') {
+                if (env.WORKSPACE) { sh script: "sudo rm -rf * .venv" }
+                checkout(
+                    scm: [
+                        $class: 'GitSCM',
+                        branches: [[name: 'refs/remotes/origin/cron_grv']],
```

Comment:
Could we change this to `origin/master`?

Reply:
Thanks for pointing it. made the changes.